src/javascript/app/pages/form/form-component.jsx

```diff
@@ -191,15 +191,15 @@ export const FormComponent = () => {
                                             options={start_dates.options}
                                             value={findTextByValue(start_dates.options, date_start)}
                                             onUpdate={updateFormField}
-                                            elementId='date_start'
+                                            element_id='date_start'
                                         />
                                     </div>
                                     {date_start !== 'now' && (
                                         <div className='form_field field-pb'>
                                             <TimePickerDropdown
                                                 time={time_start}
                                                 onUpdate={updateFormField}
-                                                elementId='time_start'
+                                                element_id='time_start'
```

Comment:
`elementId` to `element_id` in `TimePickerDropdown` as well.

Reply:
same for this I kept it `elementId`